MyCryptoHQ/MyCrypto
# Disable buttons on confirmation modal when transaction is broadcasting

Closes #1911

### Description

Fixes an issue where the confirmation modal would show clickable actions while transaction was being processed.

### Changes

* Modals now properly pass down the `disableButtons` property to their body.

### Steps to Test

1. Perform a transaction.
2. When the confirmation modal opens, click "Send".
3. The buttons should be visible and disabled.
